### PR TITLE
Webp banner Updation

### PR DESCRIPTION
**Pull request by puneethkanna**
But, there is also a problem. All the images extensions are changed to webP, need to change some of them back as Many of the post images are also converted to webP

### CODE REVIEW

**Comment by puneethkanna** [comment]
Accept the request, so we can check if it is working, and accepting should be like reverting with a just one click

File: _posts/1999-05-17-creating-browser-extensions.md [hunk 1]
```diff
@@ -130,10 +130,10 @@ Download the images <a href="https://devskrate.github.io/assets/images/browser/c
 
 ```json
 "icons": {
-      "16": "images/get_started16.png",
-      "32": "images/get_started32.png",
-      "48": "images/get_started48.png",
-      "128": "images/get_started128.png"
+      "16": "images/get_started16.webp",
```
Comment (SriSatyaLokesh) [inline _posts/1999-05-17-creating-browser-extensions.md:133]: did you change these images? to webp

Comment (puneethkanna) [reply _posts/1999-05-17-creating-browser-extensions.md:133]: Only some of the whole

**Review by SriSatyaLokesh — APPROVED** [review]

**Comment by SriSatyaLokesh** [comment]
Let's try our luck

**Pull request merged**